### DISCUSSION TITLE
4.4.0 rollout strategy

### DISCUSSION THREAD

**Discussion by Safihre** [sabnzbd/sabnzbd#2987] [General]
Last year we reverted back to using an old PyInstaller because the newer one kept getting flagged by virusscanners, like crazy. Even days after the release.
However, this release we had to update to the latest version to get Python 3.13 support.
Because I expect virusscanners will go crazy again, I am planning to release 4.4.0 at first as if it's a Testing release.
So only trigger update notifications for users that have configured Testing-releases and keeping the "Stable" version on the main and Downloads page 4.3.3.

This way enthusiasts that already use Testing releases pave the way for the broader audience, before we swap in 4.4.0 after a week or 2.

Just an FYI :)

**Comment by sanderjo** [comment]
> This way enthusiasts that already use Testing releases pave the way for the broader audience, before we swap in 4.4.0 after a week or 2.

Clever!

**Comment by Safihre** [comment]
I will switch it to 4.4.0 now, let's see how it goes.

**Reply by thezoggy** [reply]
I noticed when uninstalling sab 4.3.1 on my win11 box it warned that the uninstaller didn't have company/publisher info.
